src/lib/mapping.js

```diff
@@ -70,7 +70,8 @@ function mapTeamList (row) {
     description: row.description,
```

Comment:
@nherment @paolochiodi @mihaidma @mcdonnelldean 

I don't think we will need it but just to put it in the open: 

often in generic API response the structure is 

```
{
 data: //.... whatever you actual data is
 ... // metadata
}

// ie: 

{
  data: [{...}, {...}, {...},]
  total: 100,
  limit: 5,
  ...
}

{
  data: { id: 'myid', property: '...'}
  something: 'here',
  ...
}

```

Apart form the list endpoints, I do not see any scenario where we want to add metadata to the response. Do you think we should move to a more generic structure in the api response?